zerver/models.py

```diff
@@ -378,13 +385,10 @@ class RealmEmoji(models.Model):
     name = models.TextField(validators=[MinLengthValidator(1),
                                         RegexValidator(regex=r'^[0-9a-z.\-_]+(?<![.\-_])$',
                                                        message=_("Invalid characters in emoji name"))])  # type: Text
-    file_name = models.TextField(db_index=True, null=True)  # type: Optional[Text]
+    file_name = models.TextField(db_index=True, null=True, blank=True)  # type: Optional[Text]
```

Comment:
Why they change to add `blank=True` here?

Reply:
Otherwise `full_clean()` in [actions.py](https://github.com/zulip/zulip/blob/617aea7507f0641a46adf8ba303096db5a61a647/zerver/lib/actions.py#L4241) complains that `file_name` can't be empty.